### DISCUSSION TITLE
UI 마무리 단계 잔여일정입니다.

### DISCUSSION THREAD

**Discussion by hoospacekor** [7-7-2/DDocker_FE#41] [General]
1. 필수
- 레이아웃 통합시 **Page Content 사이즈 통일확인**(Header, Footer 간격)
- **Grid 리사이즈**

2. 미구현
- CoffeePage
  - 달력 추후 작업으로 연기
  - **카페인 정보 우선** 
- **CaffeineFilter** 컴포넌트
  - Post Page
   - **수정/삭제 오버레이**
   

3. 구체화
 - Edit페이지 인풋 추가
 - **Notification case 세분화**
 - Comments 답글 기능 구현시 더보기 추가

4. Chores
- 로그인 페이지 로고 추가
- 파비콘 추가

**Comment by hoospacekor** [comment]
### 구현
- [x] CaffeineSum.tsx
- [x] CaffeineFilter

**Reply by hoospacekor** [reply]
## 해결
레이아웃 통합시 Page Content 사이즈 통일확인(Header, Footer 간격) => 100vw

**Reply by hoospacekor** [reply]
추후 PR시 반영

**Reply by hoospacekor** [reply]
- [x] SearchPage 높이 조정
- [x] 레이아웃 통합시 Page Content 사이즈 통일확인(Header, Footer 간격) => 100vw

**Comment by cuconveniencestore** [comment]
- [x] InitilalForm.tsx 내부 ProfileIm.tsx 추가
- [x] startPage signIn.tsx size 수정

**Comment by wavescats** [comment]
- [ ] Layout (Footer) 이슈
   - 푸터가 없는 페이지(`post`) 접근 후 뒤로가기 시도 시(`/` 로 이동) 푸터 영역 사라짐 이슈 발생
   - 뒤로가기 시 이전 기록(`post` 접근 전 위치) 렌더 후 리렌더링 발생
   
(👇 이슈 영상 크기 문제로 URL 로 대체 😭)
https://github.com/7-7-2/DDocker/assets/96659041/b62fd956-ee73-4c7d-af03-0b8f249ba31a
